docs/changehistory/NextVersion.md

```diff
@@ -2,3 +2,38 @@
 publish: false
 ---
 # NextVersion
+
+Table of contents:
```

Comment:
these changes shouldbe in `4.3.0.md` since it was released in 4.3

Reply:
done